typescript/packages/well-log-viewer/src/Storybook/helpers/mui-components.tsx

```diff
@@ -0,0 +1,320 @@
+import React, { useEffect } from "react";
+
+import type { TooltipProps as MuiTooltipProps } from "@mui/material";
+import {
+    Box,
+    // eslint-disable-next-line no-restricted-imports
+    Tab,
+    // eslint-disable-next-line no-restricted-imports
+    Tabs as MUITabs,
+    Tooltip as MuiToolTip,
+    Typography,
+    tabsClasses,
+} from "@mui/material";
+
+/**
+ * Remove spaces and set to lower case.
+ * Note that we cannot use replaceAll function
+ * below until we move to ES2021.
+ */
+const buildId = (id: string): string => {
+    return id.replace(/ /g, "").toLowerCase();
+};
+
+export type TooltipProps = Pick<
```

Comment:
Can you narrow that down ?
For example, if the tooltip is not part of the pb, you can remove it (it's like 25% of this file).

Reply:
Unfortunately I couldn't get rid of tooltip - without it the tabs component collapses. I did however remove export key words in that file that I don't use in my example.